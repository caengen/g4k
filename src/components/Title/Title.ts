import styled from "styled-components";

const Title = styled.h1`
  color: white;
  text-shadow: -0.1em 0.1em 0 rgba(0,0,0,0.2);
`;

export default Title;