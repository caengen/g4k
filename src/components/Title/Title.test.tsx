import React from "react";
import { shallow } from "enzyme";
import Title from "./Title";

it("renders correctly", () => {
  const wrapper = shallow(<Title>Test</Title>);
  expect(wrapper).toMatchSnapshot();
});