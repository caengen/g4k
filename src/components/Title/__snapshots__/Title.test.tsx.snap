// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  color: white;
  text-shadow: -0.1em 0.1em 0 rgba(0,0,0,0.2);
  margin: 0;
",
        ],
      },
      "displayName": "styled.h1",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h1",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Test
</StyledComponent>
`;
